src/main/java/Duke.java

```diff
@@ -1,181 +1,18 @@
 import java.io.IOException;
-import java.util.ArrayList;
-import java.util.Scanner;
-import java.io.File;
-import java.io.FileNotFoundException;
-import java.io.FileWriter;
-import java.util.Arrays;
 
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        String greetingText = "    ____________________________________________________________\n" +
-                "     Hello! I'm Duke\n" +
-                "     What can I do for you?\n" +
-                "    ____________________________________________________________";
-        System.out.println(greetingText);
-
+        Ui.printHello();
         String filePath = "data/duke.txt";
-        Scanner sc = new Scanner(System.in);
-        ArrayList<Task> taskList = new ArrayList<Task>();
+        Storage storage = new Storage(filePath);
+        TaskList taskList;
         try {
-            loadTasks(filePath, taskList);
+            taskList = storage.getTaskList(); // initialise from text file
+            Parser.scan(taskList, storage);
         } catch (IOException e) {
```

Comment:
so, if any error is caught here, what happens? The system will be terminated. Do you think it is a good way?